### PR TITLE
Cu 7ev1hz identity overview

### PR DESCRIPTION
**Pull request by Hamza-A**
This branch aims to make `/account/logins/[type]/[id]` page as close as possible to our wireframe

## Description

Everything happens in `<IdentityLine>` component, I've added a "primary" tag if the identity is primary, otherwise there would be a "make this my primary [identity]" button displayed (only if the identity is validated), the button doesn't execute any action yet.

If the identity is not valid, there would be an "awaiting validation" tag and a "proceed to validation" button which redirects to the validation page.

At last I've added some text below each validated identity "added on..." and "last update to login on..." which contains hardcoded dotted lines instead of dates for now. 

## How Has This Been Tested?

I've added 4 tests in `tests/pages/showIdentity.test.tsx` page

## Screenshots

image
image
image

## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist:

- :green_circle: My code follows the code style of this project.
- :red_circle: My change requires a change to the documentation.
- :red_circle: I have updated the documentation accordingly.
- :green_circle: I have read the [**CONTRIBUTING**][contributing_file] document.
- :green_circle: I have added tests to cover my changes.
- :green_circle: All new and existing tests passed.

[contributing_file]: https://github.com/fewlinesco/connect/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: src/components/IdentityLine.tsx [hunk 1]
```diff
@@ -39,6 +41,28 @@ export const IdentityLine: React.FC<IdentityLineProps> = ({ identity }) => {
...
+          </Button>
+        </>
+      )}
+      {identity.primary && identity.status === "validated" && <p>Primary</p>}
+      {identity.status === "validated" && (
+        <>
+          <br />
+          <small>Added on ...</small>
+          <br />
+          <small>Last used to login on ...</small>
```
Comment (Fuzznimp) [inline src/components/IdentityLine.tsx:60]:
```suggestion
...
    <p>Added on ...<p>
    <p>Last used to login on ...pl>
...

const IdentityBox = styled.div`
  padding: ${({ theme }) => theme.spaces.component.xs};
  
  p {
 	margin 
  }
`;
```

**Review by Fuzznimp — CHANGES_REQUESTED** [review]

File: src/components/IdentityLine.tsx (hunk 1 above)
Comment (Hamza-A) [reply src/components/IdentityLine.tsx]: OK fixed! 4f42144

File: src/components/IdentityLine.tsx [hunk 2]
```diff
@@ -47,6 +69,13 @@ const IdentityBox = styled.div`
   padding: ${({ theme }) => theme.spaces.component.xs};
 `;
 
+const IdentityInfo = styled.div`
+  p {
+    font-size: 70%;
```
Comment (Fuzznimp) [inline src/components/IdentityLine.tsx:74]: You should use theme value, and more importantly never use percentages for font size. We use rem for font sizes to handle zooming in the browser (`cmd +`/ `cmd -`)

Comment (Hamza-A) [reply src/components/IdentityLine.tsx]: OK fixed fdd8618

**Review by Fuzznimp — APPROVED** [review]

**Pull request merged**